### PR TITLE
Db merge

### PR DESCRIPTION
**Pull request by corinneteeter**
Luke, I did not get the timestamping working.  line 510 of fit_average_first_pulse.py.  I also didn't comment the update_DB function because it is based on code that you wrote that I don't know the details of.  i.e. the parallelization.  The parallelization code is in there but commented out.  Also note the resulting table is in synphys11 already.

### CODE REVIEW

File: multipatch_analysis/fit_average_first_pulse.py [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+            All units in SI.""",
+            ('pair_id', 'pair.id', 'The ID of the entry in the pair table to which these results apply', {'index': True}),
+
+            # current clamp
+            ('ic_amp', 'float', 'fit amplitude of current clamp average first pulses'),
+            ('ic_latency', 'float', 'fit time elapsed since the time of presynaptic spike (max dv/dt) of current clamp data'),
+            ('ic_rise_time', 'float', 'fit rise time of psp of current clamp data'),
+            ('ic_decay_tau', 'float', 'fit decay of psp of current clamp data'),
+            ('ic_avg_psp_data', 'array', 'array of the data voltage waveform used in fitting; starts 10 ms before pre-synaptic spike'),
+            ('ic_avg_psp_fit', 'array', 'fit array of the best fit voltage waveform starting 10 ms before pre-synaptic spike'),
```
Comment (campagnola) [inline multipatch_analysis/fit_average_first_pulse.py]: Is this necessary, given how easy it should be to generate from the fit parameters?

File: multipatch_analysis/fit_average_first_pulse.py [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+            ('ic_weight', 'array', 'weighting used during fitting of current clamp data'),
+
+            # voltage clamp
+            ('vc_amp', 'float', 'fit amplitude of voltage clamp average first pulses'),
+            ('vc_latency', 'float', 'fit time elapsed since the time of presynaptic spike (max dv/dt) of voltage clamp data'),
+            ('vc_rise_time', 'float', 'fit rise time of psp measured in voltage clamp'),
+            ('vc_decay_tau', 'float', 'fit decay of psp measured in voltage clamp'),
+            ('vc_avg_psp_data', 'array', 'array of the data current waveform used in fitting; starts 10 ms before pre-synaptic spike'),            
+            ('vc_avg_psp_fit', 'array', 'fit array of the best fit current waveform starting 10 ms before pre-synaptic spike'),
+            ('vc_dt', 'float', 'time step of *avg_psp* array from voltage clamp data'),
```
Comment (campagnola) [inline multipatch_analysis/fit_average_first_pulse.py]: Maaaybe I would rather stick to using the DB default sample rate for now..

File: multipatch_analysis/fit_average_first_pulse.py [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+            ('vc_pulse_ids', 'object', 'data base pulse ids included in the voltage clamp fit'),
+            ('vc_NRMSE', 'float', 'error of fit of voltage clamp fit'),
```
Comment (campagnola) [inline multipatch_analysis/fit_average_first_pulse.py]: lower case, to follow: https://github.com/aiephys/multipatch_analysis/blob/master/multipatch_analysis/connection_strength.py#L99

File: multipatch_analysis/fit_average_first_pulse.py (hunk 1 above)
Comment (corinneteeter) [reply multipatch_analysis/fit_average_first_pulse.py]: Maybe not but it is convenient so I like it.

File: multipatch_analysis/fit_average_first_pulse.py (hunk 2 above)
Comment (corinneteeter) [reply multipatch_analysis/fit_average_first_pulse.py]: This dt is what is produced by your functions.  I am not sure it is always the same recorded it for the sake of sanity.

**Comment by corinneteeter** [comment]
Have at it.

**Pull request merged**